src/Database/Install.php

```diff
@@ -108,9 +108,9 @@ public function installConfiguration()
         $createButtonLabel = [];
 
         foreach (Language::getLanguages() as $lang) {
-            $pageName[$lang['id_lang']] = $this->translator->trans('My wishlists', [], 'Modules.BlockWishList', $lang['locale']);
-            $defaultName[$lang['id_lang']] = $this->translator->trans('My wishlist', [], 'Modules.BlockWishList', $lang['locale']);
-            $createButtonLabel[$lang['id_lang']] = $this->translator->trans('Create new list', [], 'Modules.BlockWishList', $lang['locale']);
+            $pageName[$lang['id_lang']] = $this->translator->trans('My wishlists', [], 'Modules.Blockwishlist.Admin', $lang['locale']);
+            $defaultName[$lang['id_lang']] = $this->translator->trans('My wishlist', [], 'Modules.BlockWishList.Admin', $lang['locale']);
+            $createButtonLabel[$lang['id_lang']] = $this->translator->trans('Create new list', [], 'Modules.BlockWishList.Admin', $lang['locale']);
```

Comment:
Watch out not to use CamelCase in domain components. It won't work (see [docs](https://devdocs.prestashop.com/1.7/modules/creation/module-translation/new-system/#translation-domain))

```suggestion
            $pageName[$lang['id_lang']] = $this->translator->trans('My wishlists', [], 'Modules.Blockwishlist.Admin', $lang['locale']);
            $defaultName[$lang['id_lang']] = $this->translator->trans('My wishlist', [], 'Modules.Blockwishlist.Admin', $lang['locale']);
            $createButtonLabel[$lang['id_lang']] = $this->translator->trans('Create new list', [], 'Modules.Blockwishlist.Admin', $lang['locale']);
```